src/layouts/settingPage.js

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react"
+import { Box } from '@mui/material'
+import { UserAccountSection } from "../components/setting/userAccountSection"
+import { SettingsGroup } from "../components/setting/settingGroup"
+import { SettingsItem } from "../components/setting/settingItem"
+import { AppBar } from "../components/AppBar"
+
+export default function SettingsPage() {
+  const [isLoggedIn, setIsLoggedIn] = useState(false)
+  const [darkMode, setDarkMode] = useState(false)
+  const [notifications, setNotifications] = useState(true)
+  const [newPurchase, setNewPurchase] = useState(false)
+
+  const handleLogin = () => setIsLoggedIn(true)
+  const handleLogout = () => setIsLoggedIn(false)
+
+  return (
+      <Box sx={{ minHeight: '100vh', bgcolor: 'background.default' }}>
+        <AppBar title="تنظیمات" />
+        
+        <UserAccountSection
+          isLoggedIn={isLoggedIn}
+          userData={{
+            name: "سجاد سلطانیان",
+            phone: "۹۸۵ ۷۷۲ ۷۸۲۷"
+          }}
```

Comment:
why is the user data hard coded?

Reply:
temp test.
will be removed after completing the login process.